Gopkg.lock

```diff
@@ -7,17 +7,23 @@
   packages = ["."]
   revision = "d5467c17e7afe8d8f08f556c6c811a50c3feb28d"
 
+[[projects]]
+  name = "github.com/davecgh/go-spew"
+  packages = ["spew"]
+  revision = "346938d642f2ec3594ed81d874461961cd0faa76"
+  version = "v1.1.0"
```

Comment:
Why is `spew` being used?  I'm guessing there's some uncommitted code locally that `dep` picked up.

Reply:
I believe this is actually part of Errwrap dependencies - when we remove errwrap, it will go away

Reply:
Oh maybe not....